### ISSUE TITLE
Migration missing on 3.3.1

### ISSUE THREAD

**Issue by carchan1** [pinax/django-user-accounts#374]
**Describe the bug**
There is a migration missing on `django-user-accounts` side for version 3.3.1

**Reproduce behavior**
Run 
```
python manage.py makemigration
```

Results:
```
Migrations for 'account':
  C:\my_project\venv\lib\site-packages\account\migrations\0009_alter_signupcode_max_uses.py
    - Alter field max_uses on signupcode
```

**Expected behavior**
I expect that  `django-user-accounts` comes with all its migrations

**Comment by uhurusurfa** [comment]
Something is not right - the app only has files up to prefix 0005 so not sure where your 0009 prefix file comes from.
Did you use a fork of this app previously?

**Comment by carchan1** [comment]
@uhurusurfa No, it's just that I played a bit with my DB and ran makemigrations few times 🙃 
![image](https://github.com/pinax/django-user-accounts/assets/28656858/df7253ff-6a6d-49cf-8c72-662f632c7187)

**Comment by taialex1** [comment]
It seems [this commit](https://github.com/pinax/django-user-accounts/commit/09ca7dd9326b375bcedb6db3862ce930419246f4) is missing a migration for the change of the default value.

**Comment by blag** [comment]
Possibly fixed in #375.

Perhaps we should include some CI checks that use `makemigrations` to ensure migrations are included in PRs.

And another one to ensure that all required files are included in wheels and sdists.
